### ISSUE TITLE
Static property of class in another assembly is NULL

### ISSUE THREAD

**Issue by knurdnerd** [mkaring/ConfuserEx#212]
I'm currently evaluating the project to be used as a replacement for a commercial tool and I came across an issue that seems like a bug to me.

Consider the following most simple project (I attached the full solution to this bug report):
- A .NET Framework 4.7.2 console application with a .NET Standard class library with one class only.
- Confuser.Core 1.4.1+5d92e25e43

The console application's Main:

```
        private static void Main(string[] args)
        {
            if (ClassWithConstants.StringStaticProperty == null)
            {
                throw new Exception($"{nameof(ClassWithConstants.StringStaticProperty)} is NULL");
            }

            Console.WriteLine("Everything worked fine. Press Enter to exit.");
            Console.ReadLine();
        }
```

and the class from the class library:

```
    public class ClassWithConstants
    {
        public static string StringStaticProperty => "AnotherSecretString";
    }

```

Settings for Confuser:
- everything turned off but 'constants'

It works when the class is part of the main project. Also, there is some code when I disassemble the class library, so it doesn't just skip it.

When I confuse the two output files and start the exe, it crashes with the exception of the NULL check.

The Visual Studio 2019 solution to reproduce the issue. The Confuser settings file is included in the archive.
[ReproduceConfuserBug.zip](https://github.com/mkaring/ConfuserEx/files/5173244/ReproduceConfuserBug.zip)

**Comment by KvanTTT** [comment]
Have you tried the latest version from CI? https://ci.appveyor.com/project/mkaring/confuserex/branch/master/artifacts

There are some fixes related to constants obfuscating #193 and not only.

**Comment by knurdnerd** [comment]
Thank you for the quick reply @KvanTTT ! I just tried with the latest from CI and it works fine. I'll close the bug.
